src/commands/math.js

```
@@ -306,6 +306,11 @@ var MathCommand = P(MathElement, function(_, super_) {
  */
 var Symbol = P(MathCommand, function(_, super_) {
   _.init = function(ctrlSeq, html, text, mathspeak) {
+    this.isSymbol = true;
+    if (/^\\.*[A-Za-z]$/.test(ctrlSeq)) {
+      console.error('Creating a symbol and got ctrlSeq of', ctrlSeq, 'which is missing a trailing space.');
```

I don't think console.error() fixes my concern about console.warn(). My point here is that we either plan on leaning on this at runtime as a last minute fix or we don't. If we do, then we probably shouldn't be polluting the console with expected errors. I meant we should either throw a hard error or don't log anything at all.

---

I agree that this is a programming error, and we should throw when we encounter it to force the author to fix the error, rather than either trying to fix it for them silently or issuing a warning.